src/golang/lib/collections/operator/postgres.go

```diff
@@ -24,6 +24,39 @@ func newPostgresWriter() Writer {
 	return &postgresWriterImpl{standardWriterImpl{}}
 }
 
+func (r *postgresReaderImpl) ObjectTouchedByWorkflow(
```

Comment:
In a second thought I feel we could use 'ObjectModified' to emphasize those are 'written'. 'Touched' could sounds for objects both read and written by the workflow.

Also consider adding a comment describing we are returning all loads operators with the given object name. Otherwise one might wonder why we are returning a list of objects here.

Reply:
Renamed to `GetLoadOperatorsForWorkflowAndIntegration`